app/views/otherbusinessinvolvements/HaveVatNumber.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.data.Form
+@import play.api.mvc.Request
+@import play.api.i18n.Messages
+@import config.FrontendAppConfig
+
+@this(
+    layout: layouts.layout,
+    h1: components.h1,
+    p: components.p,
+    button: components.button,
+    formWithCSRF: FormWithCSRF,
+    errorSummary: components.errorSummary,
+    yesNoRadio: components.yesNoRadio
+)
+
+@(haveVatNumberForm: Form[Boolean], index: Int)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@layout(pageTitle = Some(title(haveVatNumberForm, messages("obi.haveVatNumber.heading"))), backLink = true) {
+
+    @errorSummary(errors = haveVatNumberForm.errors)
+
+    @h1("obi.haveVatNumber.heading")
+
+    @formWithCSRF(action = controllers.registration.otherbusinessinvolvements.routes.HaveVatNumberController.submit(index)) {
+
+        @yesNoRadio(
+            form = haveVatNumberForm,
+            headingKey = messages("obi.haveVatNumber.heading"),
+            isPageHeading = false,
```

Comment:
`isPageHeading` should be set to `true` and the h1 on line 38 can be removed. 

This is because there is no content between the heading and the form, so the h1 should be inside the radio group's fieldset legend for accessibility purposes.